test/e2e/parallel/registration_service_test.go

```diff
@@ -581,6 +582,136 @@ func TestPhoneVerification(t *testing.T) {
 	require.NotEmpty(t, otherUserSignup.Annotations[toolchainv1alpha1.UserSignupVerificationCodeAnnotationKey])
 }
 
+func TestActivationCodeVerification(t *testing.T) {
+	// given
+	t.Parallel()
+	await := WaitForDeployments(t)
+	hostAwait := await.Host()
+	route := hostAwait.RegistrationServiceURL
+	// hostAwait.UpdateToolchainConfig(testconfig.AutomaticApproval().Enabled(true)) // user is automatically approved if the activation code is valid (ie, `verification-required` state is removed)
```

Comment:
before commenting out this line (which I should have removed, btw), I got multiple errors in others e2e tests which are run in parallel of this one (`test/e2e/parallel`) and expected that their own `UserSignup` resource is in `PendingApproval` status.

Reply:
I removed the commented out code in ef7601a

Reply:
ok so I guess that some tests require automatic approval to be off and since these tests are run in parallel in the same environment they would all share the same ToolchainConfig so I guess that's why we cannot change it